docs/source/contributing/developing.adoc

```diff
@@ -131,10 +131,9 @@ If you seeing Glide errors during `make vendor`, you can clear the Glide cache v
 == Building Minishift
 
 [[build-minishift-binary]]
-=== Building the Minishift binary
+=== Building the Minishift Binary
```

Comment:
@thatdocslady I confused why `the` has small `t` but other words have capital letter as starting letter.